packages/wdio-browserstack-service/src/instrumentation/funnelInstrumentation.ts

```diff
@@ -127,6 +127,9 @@ function buildEventData(eventType: string, config: BrowserStackConfig): any {
         eventProperties.productUsage = getProductUsage(workerData)
         eventProperties.isPercyAutoEnabled = config.isPercyAutoEnabled
         eventProperties.percyBuildId = config.percyBuildId
+        if (process.env[BSTACK_A11Y_POLLING_TIMEOUT]) {
+            eventProperties.allyPollingTimeout = process.env[BSTACK_A11Y_POLLING_TIMEOUT]
```

Comment:
`pollingTimeout`, this is the name in Java.. So, let's keep consistent.

Reply:
Done